online/failover.go

```diff
@@ -41,3 +41,16 @@ func (c *client) DeleteMACFailoverIP(address string) error {
 
 	return nil
 }
+
+func (c *client) SetReverseFailoverIP(address, reverse string) error {
+	target := fmt.Sprintf("%s/server/edit", serverEndPoint)
```

Comment:
`/server/ip/edit` would be the right endpoint here

Reply:
just changed, sorry about this slip!

Reply:
Seems to be the same here :confused:

Reply:
Strange, but now it should be added in properly 😁